src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -112,6 +112,10 @@ def get(self, request, organization):
             "organizations:dynamic-issue-counts", organization, actor=request.user
         )
 
+        include_inbox = features.has(
+            "organizations:inbox", organization, actor=request.user
+        ) and "group_inbox" in request.GET.get("expand", [])
```

Comment:
Should this be `request.GET.getlist()` Using .get() could get you a string if there is only one expand defined. Querystring parameters can be defined like `expand=one&expand=two` and `getlist()` will work consistently while `get()` will not from what I remember.